dialoptions.go

```diff
@@ -736,16 +741,19 @@ func WithIdleTimeout(d time.Duration) DialOption {
 	})
 }
 
-// WithMaxRetryAttempts returns a DialOption that configures the maximum number
+// WithMaxCallAttempts returns a DialOption that configures the maximum number
 // of retry attempts for the channel. Service owners may specify a higher value
```

Comment:
How about:

`...configures the maximum number of attempts per call (including retries and hedging) using the channel`?

I don't especially like mentioning "hedging" here since we don't support it yet, but it seems better than forgetting it later...